opencti-platform/opencti-graphql/src/http/httpServer.js

```diff
@@ -93,13 +93,41 @@ const createHttpServer = async () => {
s4jnGC8iunlEdAAAAAAAAAAAgIIpXkAnnptFPEe+ENHNI6IDAAAAAAAAAAAURHECOvHcLOI58o2Ibh4RHQAAAAAAAAAAIO8KH9CJ52YRz1EoRHTziOgAAAAAAAAAAAB5VdiATjw3i3iOQjt00SuSdZWI6OYQ0QEAAAAAAAAAAPKmcAGdeG4W8RzF0jl3IxHd9DYgogMAAAAAAAAAAORDYQI68dws4rl5MzrWqLn9hxVzJCGiB2AbENEBAAAAAAAAAADGK/8BnXhuFvHcvOb2lbL8RyUtVXP7zyrmaEJED8A2IKIDAAAAAAAAAACMR34DOvHcLOK5ec3tKyU9Lsk68ZVFRPRAIaKbR0QHAAAAAAAAAACBlb+ATjw3i3hu3jvjeQ4RPViI6OYR0QEAAAAAAAAAQCDlJ6ATz80inps3dDzPIaIHCxHdPCI6AAAAAAAAAAAInPEHdOK5WcRz884ez3OI6MFCRDePiA4AAAAAAAAAAAJlfAGdeG4W8dy8kcfzHCJ6sBDRzSOiAwAAAAAAAACAwBh7QCeem0U8N2/08TyHiB4sRHTziOgAAAAAAAAAACAQxhbQiedmEc/NG3s8zyGiBwsR3TwiOgAAAAAAAAAAMG70AZ14bhbx3Lzxx/McInqwENHNI6IDAAAAAAAAAACjRhfQiedmEc/Ny188zyGiBwsR3TwiOgAAAAAAAAAAMGbkAZ14blZpzJ94PjZE9GAhoptHRAcAAAAAAAAAAEaMLKATz81i/uYVLp7nENGDhYhuHhEdAAAAAAAAAAAU3dkDOvHWLOZvXuHjeQ4RPViI6OYR0QEAAAAAAAAAQFENH9CJt2Yxf/OKF89ziOjBQkQ3j4gOAAAAAAAAAACKZuiATrw1i/mbV/x4nkNEDxYiunlEdAAAAAAAAAAAUBRnDujEW7OYv3nm4nkOET1YiOjmEdEBAAAAAAAAAEDBvTOgE2/NYv7mmY/nOUT0YCGim0dEBwAAAAAAAAAABfXbAZ14axbzNy848TyHiB4sRHTziOgAAAAAAAAAAKBgTgZ04q1ZzN+84MXzHCJ6sBDRzSOiAwAAAAAAAACAghgM6MRbs5i/ecGN5zlE9GAhoptHRAcAAAAAAAAAAHk3GNDt0LMi3hrcCszfqHP3vEfBjuc5i9Tc/lDFHJ06526U51+nYEf0R8t7G8xbIumnAf4Jv6Dpu85hKQcAAAAAAAAAAPly4hLuoZskxQL485V/vGX+5h2+8HVJz5bAT9or17+/oo5Qh+f/4sSZ6JkA/nQHNNB3d9lvg855Nyi4Z6J/Skcu6mIpBwAAAAAAAAAA+TIY0DvPj0mhuQpWxK2QeM78g7EN5t0q6UcB/gl75foX6cj8wxV3lBq8nPuHFayIfkADffMUvzxZIftHEC/n/kl1zvsqyzgAAAAAAAAAAMgn++1/C1bErax4m5t/WOdJ6mb+prZBYCN65cbzt7dNoCJ6ZcXzk/tHkCI68RwAAAAAAAAAABSE/Vv/KxgRvTLjrSS9Ma9XYc2V2YheufOXghjRiedvb5tARPTKjOcn948gRHTiOQAAAAAAAAAAKBj7HV8xG9ErO95KpiM685eCFNGJ5+/YNkYj+sGKjucn9w+TEZ14DgAAAAAAAAAACso+41fNRHTibY6ZiM78f2sfMB7RiedDbhsjEf2gBvrmVnw8P7l/mIjoxHMAAAAAAAAAAFBw9pD/pbgRnXh7uuJGdOZ/xn3AWEQnnp912xQ1ohPPz7x/FDOiE88BAAAAAAAAAEBR2MP+1+JEdOLtUIoT0Zn/sPtA0SM68XzE26YoEZ14Pvz+UYyITjwHAAAAAAAAAABFY5/1TxQ2ohNvz6awEZ35j0TxIjrxfNTbpqARnXg+sv2jkBGdeA4AAAAAAAAAAIrKHtGfKkxEJ96OVGEiOvMfjcJHdOL5mLdNQSI68Xx0+0chIjrxHAAAAAAAAAAAFJ01qj/dvK9RynZIahzn/y/xdizmtDcoow5J05i/Ic3tP5S0NM/flXiel23T8XuS/wtJ4XF+J+L52PePZyXdkofvRDwHAKAUnPPim7Jqm4z/HF7/IXVfN5cNAgAAAAAA8sEa9d8Yf0Qn3o7H+CM68x+v5vYfSLo1T9+NeJ7XbTPuiE48H//+Md6ITjwHAAAAAAAAAADGWGP6W2OP6MTbfBh7RGf++ZKfiE48L8i2GXNEJ57nb/8Ya0QnngMAAAAAAAAAAKOsMf/N0Ud04m0+jT6iM/98G19EJ54XdNuMOqITz/O/f4w2ohPPAQAAAAAAAACAcda4/vbIIzrxthBGHtGZf6GMLaITz4uybUYc0Ynnhds/RhrRiecAAAAAAAAAACAQrHF/h7NHdOJtIZ09ojP/QhtdRCeeF3XbnDWiE88Lv3+cLaITzwEAAAAAAAAAQGBYefkuQ0d04m0xDB3RmX+xjCyiE8+NbJshIzrxvHj7x1ARnXgOAAAAAAAAAAACxc7Ld+k8PyaF5kqKnfJV4m2xvDGvV2HNldR9yldTsu0rmH+RdM67TdIPh/kTxHNj22buRsn6sKTMKV8lnhd3/1gi6bnTvko8BwAAAAAAAAAAgWPl9budPBO9VsTz4jt5Jvok2fYHdODCNoZSZGc+E514Hoht8/aZ6F3Ec2P7R+5MdOI5AAAAAAAAAACoEM37GtW8ey6DYP6Vuw3af6Dmdv/EP8c0ffe5DCUgZu5aoMhrNQzC5P7RcQVDAAAAAAAAAAAAACrJYEQnngMAAAAAAAAAAAAAwJnOAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACgRP1/czqtTs2HjBMAAAAASUVORK5CYII=" alt="OpenCTI" width="250" style="vertical-align: middle; clear: both; width: 250px; max-width: 250px; padding-top: 40px; padding-bottom: 40px;"></div>
```

Comment:
We need to find a way to use the already define logo and not use it directly in base64

Reply:
Even better to create a simple page that will redirect auto?

Reply:
we'll do an automatic redirection then :)